ui/components/app/edit-gas-display/edit-gas-display.component.js

```diff
@@ -107,6 +108,11 @@ export default function EditGasDisplay({
       dappSuggestedAndTxParamGasFeesAreTheSame,
   );
 
+  let warningMessage;
+  if (gasLimit < properGasLimit) {
```

Comment:
This comparison should be done using `BigNumber`

Reply:
@danjm I modified this comparison by using `BigNumber` as you suggested.
You can see changes here: 8e8a6ecf4a162029c09ca4616d393bde27dc713a
Is it okay now?